src/main/java/com/google/finapp/ArgsModule.java

```diff
@@ -91,6 +95,9 @@ String provideSpannerDatabaseId() {
 
     @Parameter(names = {"--spanner_database_id"})
     String spannerDatabaseId;
+
+    @Parameter(names = {"--spanner_dao_impl"})
+    String spannerDaoImpl;
```

Comment:
Since we only have two options, I think we should use a boolean value here. [See this documentation](https://jcommander.org/#_boolean). With this, we can rename the parameter to something like `useJDBC` and have the default be set to `false`. Thus, no actual arguments are needed (just the flag by itself), and we don't have to check for any strings like "jdbc" or "client".